checklists/alz_checklist.en.json

```diff
@@ -1370,15 +1339,14 @@
             "service": "Firewall",
             "guid": "22d6419e-b627-4d95-9e7d-019fa759387f",
             "id": "D07.11",
-            "ammp": true,
             "severity": "High",
             "graph": "resources | where type=='microsoft.network/virtualnetworks' | project id,subnets=properties.subnets | mv-expand subnets | project id, subnetName = subnets.name, subnetPrefix = subnets.properties.addressPrefix | extend subnetPrefixLength = split(subnetPrefix, '/')[1] | where subnetName == 'AzureFirewallSubnet' | extend compliant = (subnetPrefixLength == 26) | distinct id, compliant",
             "link": "https://learn.microsoft.com/azure/firewall/firewall-faq#why-does-azure-firewall-need-a--26-subnet-size"
         },
         {
             "category": "Network Topology and Connectivity",
             "subcategory": "Firewall",
-            "text": "Arrange rules within the firewall policy into Rule Collection Groups and Rule Collections and based on their frequency of use",
+            "text": "[Need to determine what this is really saying]Arrange rules within the firewall policy into Rule Collection Groups and Rule Collections and based on their frequency of use",
```

Comment:
Agree, there's been some content published around this recently which we could review and see if it makes sense to add here. Talks about how to design rule collection groups

Reply:
I know of some blog posts, but is there some Learn content we can pull from?